<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterHighlighting</class>
 <widget class="QWidget" name="CharacterHighlighting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>456</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout" name="fontLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QHBoxLayout" stretch="0,0,0,0,1">
       <item>
        <widget class="QCheckBox" name="enableText"/>
       </item>
       <item>
        <widget class="QLabel" name="textColorLabel">
         <property name="text">
          <string>Text color:</string>
         </property>
         <property name="buddy">
          <cstring>textColor</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KColorButton" name="textColor"/>
       </item>
       <item>
        <widget class="QToolButton" name="resetTextColor">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" stretch="0,0,0,0,1">
       <item>
        <widget class="QCheckBox" name="enableBackground">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="backgroundColorLabel">
         <property name="text">
          <string>Background color:</string>
         </property>
         <property name="buddy">
          <cstring>backgroundColor</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KColorButton" name="backgroundColor"/>
       </item>
       <item>
        <widget class="QToolButton" name="resetBackground">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout">
     <item row="0" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Underlining:</string>
       </property>
       <property name="buddy">
        <cstring>underlineStyle</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="KComboBox" name="underlineStyle"/>
       </item>
       <item>
        <widget class="KComboBox" name="underlineLineStyle">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KColorButton" name="underlineColor">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Strikethrough:</string>
       </property>
       <property name="buddy">
        <cstring>strikethroughType</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QComboBox" name="strikethroughType"/>
       </item>
       <item>
        <widget class="QComboBox" name="strikethroughLineStyle">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KColorButton" name="strikethroughColor">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Capitalization:</string>
       </property>
       <property name="buddy">
        <cstring>capitalizationList</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="capitalizationList"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel">
       <property name="text">
        <string comment="Character position">Position:</string>
       </property>
       <property name="buddy">
        <cstring>positionList</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="positionList"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
